Tests/Repositories_Test/RepositoriesTest.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using Xunit;
+using System.Collections.Generic;
+using System.Data.SQLite;
+using System.Threading;
+using Microsoft.EntityFrameworkCore;
+using OLA2_SofQuality.Data;
+using OLA2_SofQuality.Models;
+using OLA2_SofQuality.Repositories;
+
+
+namespace OLA2_SofQuality.Tests.Repositories_Test;
+
+public class RepositoriesTest : IDisposable
+{
+    private readonly TaskRepository _taskRepository;
+    private readonly SQLiteConnection _connection;
+
+    public RepositoriesTest()
+    {
+        var connectionString = "Data Source=:memory:;Version=3;New=True;";
```

Comment:
Godt at se du fik in memory til at virke! Håber du havde en mere smooth implementation end jeg havde